src/discordcr/mappings/permissions.cr

```diff
@@ -0,0 +1,28 @@
+module Discord
+  enum DISCORD_PERMISSIONS
```

Comment:
The convention for enum names and members is the same as other types: CamelCase.